flash/pointcloud/segmentation/open3d_ml/app.py

```diff
@@ -50,8 +50,8 @@ def visualize_dataset(self, dataset, split, indices=None, width=1024, height=768
         # Setup the labels
         lut = LabelLUT()
         color_map = dataset.color_map
-        for id, val in dataset.label_to_names.items():
-            lut.add_label(val, id, color=color_map[id])
+        for idx, val in dataset.label_to_names.items():
+            lut.add_label(val, idx, color=color_map[idx])
```

Comment:
If: https://github.com/PyTorchLightning/lightning-flash/pull/1212#discussion_r820202531

```suggestion
        for id_color, val in dataset.label_to_names.items():
            lut.add_label(val, id_color, color=color_map[idx])
```